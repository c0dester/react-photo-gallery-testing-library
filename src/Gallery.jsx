import React from "react";
import RPhG from "react-photo-gallery";

const photos = new Array(5).fill(undefined).map((item, index) => ({
  src: `https://picsum.photos/600/800?random=${index}`,
  height: 3,
  width: 4
}));

export default function Gallery() {
  return (
    <div data-testid="gallery">
      <RPhG photos={photos} />
    </div>
  );
}
