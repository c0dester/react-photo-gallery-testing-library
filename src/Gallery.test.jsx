import "@testing-library/jest-dom/extend-expect";
import React from "react";
import { render } from "@testing-library/react";
import Gallery from "./Gallery";

test("Test Gallery", () => {
  const { asFragment } = render(<Gallery />);
  expect(asFragment()).toMatchSnapshot();
});
