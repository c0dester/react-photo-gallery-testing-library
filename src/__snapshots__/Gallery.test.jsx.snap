// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Gallery 1`] = `
<DocumentFragment>
  <div
    data-testid="gallery"
  >
    <div>
      Â 
    </div>
  </div>
</DocumentFragment>
`;
